src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -674,4 +673,7 @@
   <ItemGroup>
     <None Include="Resources\SR.resx" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Cryptography.OpenSsl\src\System.Security.Cryptography.OpenSsl.csproj" />
```

Comment:
OpenSSL is a Unix only project, can this cause any issues on non-Unix?

Reply:
We have a platform independent configuration in openssl